ArduCopter/Parameters.cpp

```diff
@@ -664,7 +664,7 @@ const AP_Param::Info Copter::var_info[] = {
     GOBJECT(optflow,   "FLOW", AP_OpticalFlow),
 #endif
 
-#if PRECISION_LANDING == ENABLED
+#if AC_PRECLAND_ENABLED == ENABLED
```

Comment:
```suggestion
#if AC_PRECLAND_ENABLED
```
This is our poattern for all of these new defines.  Hopefully at some stage we can get rid of `ENABLED` and `DISABLED` - they're in the global namespace and cause all sorts of problems.